client/utils/navigation.js

```diff
@@ -1,11 +1,6 @@
-import { NamedURLResolver } from 'react-router-named-routes'
-import { browserHistory } from 'react-router'
+import { history } from 'base'
```

Comment:
Again, I think we can continue to use `browserHistory` from react-router instead of using the history library directly.  Those two libraries change at different rates, so using the version that react-router uses internally gives us a better compatibility story, IMO.